Mathlib/CategoryTheory/Subpresheaf/Subobject.lean

```diff
@@ -0,0 +1,76 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Subpresheaf.Image
+import Mathlib.CategoryTheory.Subobject.Basic
+
+/-!
+# Comparison between `Subpresheaf`, `MonoOver` and `Subobject`
+
+Given a presheaf of types `F : Cᵒᵖ ⥤ Type w`, we define an equivalence
+of categories `Subpresheaf.equivalenceMonoOver F : Subpresheaf F ≌ MonoOver F`
+and an order isomorphism `Subpresheaf.orderIsoSubject F : Subpresheaf F ≃o Subobject F`.
+
+-/
+
+universe w v u
+
+namespace CategoryTheory
+
+variable {C : Type u} [Category.{v} C] (F : Cᵒᵖ ⥤ Type w)
+
+namespace Subpresheaf
+
+/-- The equivalence of categories `Subpresheaf F ≌ MonoOver F`. -/
+@[simps]
+noncomputable def equivalenceMonoOver : Subpresheaf F ≌ MonoOver F where
+  functor :=
+    { obj A := MonoOver.mk' A.ι
+      map {A B} f := MonoOver.homMk (Subpresheaf.homOfLe (leOfHom f)) }
+  inverse :=
+    { obj X := Subpresheaf.range X.arrow
+      map {X Y} f := homOfLE (by
+        dsimp
+        rw [← MonoOver.w f]
+        apply range_comp_le ) }
+  unitIso := NatIso.ofComponents (fun A ↦ eqToIso (by simp))
+  counitIso := NatIso.ofComponents
+    (fun X ↦ MonoOver.isoMk ((asIso (toRange X.arrow)).symm))
+
+variable {F} in
+@[simp]
+lemma range_subobject_mk_ι (A : Subpresheaf F) :
```

Comment:
I think the naming convention wants this to be called `range_subobjectMk_ι`. Similar for the lemma below.

Reply:
Thanks!